components/Post.vue

```diff
@@ -68,7 +82,7 @@ figure {
   width: 100%; /* Set a small width */
   height: 60%;
   display: inline-block;
-  background-position: top;
+  background-size: cover;
```

Comment:
@TeddTech I think as per @jobara's comment you should also remove this change from this PR as it is now addressed by #35.

Reply:
This change has been completed